locale/en/BlueprintTools.cfg

```diff
@@ -60,3 +66,5 @@ bpt-blueprint-absolute-snapping-disabled=Blueprint does not use absolute grid sn
 bpt-blueprint-book-absolute-snapping-disabled=Selected book blueprint does not use absolute grid snapping
 bpt-blueprint-snap-to-grid-disabled=Blueprint does not use grid snapping
 bpt-unable-to-clear-cursor=Unable to clear cursor
+bpt-unable-to-add-filter=Unable to add filter
```

Comment:
Could you add more detailed errors, such as "filter already exists"? The generic one isn't very descriptive.
![image](https://github.com/user-attachments/assets/e315419b-2f21-48f3-9dff-c6bd1d53830b)

Reply:
It took a little bit of restructuring, but the warning messages are much more comprehensive now.